extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/ResteasyReactiveServerRuntimeConfig.java

```diff
@@ -0,0 +1,38 @@
+package io.quarkus.resteasy.reactive.server.runtime;
+
+import java.nio.charset.Charset;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "resteasy-reactive", phase = ConfigPhase.RUN_TIME)
+public class ResteasyReactiveServerRuntimeConfig {
+
+    /**
+     * Input part configuration.
+     */
+    @ConfigItem
+    public MultipartConfigGroup multipart;
+
+    @ConfigGroup
+    public static class MultipartConfigGroup {
+
+        /**
+         * Input part configuration.
+         */
+        @ConfigItem
+        public InputPartConfigGroup inputPart;
+    }
+
+    @ConfigGroup
+    public static class InputPartConfigGroup {
```

Comment:
Does it really need this extra layer of config? It makes for a very long config property.

Reply:
The reason I did it was to emulate what we have in RESTEasy Classic. I wanted the full config name to be the same (with the only change being swapping `resteasy` for `resteasy-reactive`).